includes/Commerce/Orders.php

```diff
@@ -40,8 +40,21 @@ class Orders {
 	 */
 	public function find_local_order( $remote_id ) {
 
-		// TODO: implement
-		return null;
+		$order_id_array = get_posts( [
```

Comment:
How about we use [`wc_get_orders()`](https://github.com/woocommerce/woocommerce/wiki/wc_get_orders-and-WC_Order_Query)? That way we'll have less to update once WC switches away from custom post types. It seems to still support the same meta query args.

Reply:
Updated in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1514/commits/b6e5820a5b270a5cea1b187a24eca56984c80bea